tests/analyse.test.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env jest */
+import analyse from '../src/analyse';
+
+test('it returns an object with the average,minimum, maximum and length of the array', () => {
+  const object = analyse([1, 8, 3, 4, 2, 6]);
+  expect(object).toEqual({
+    average: 4,
+    min: 1,
+    max: 8,
+    length: 6,
+  });
+});
```

Comment:
**[ OPTIONAL ]** For this test we could improve your testing by dividing it in different assertions and wrapping them in a describe block, which would indicate the scope of the testings followed by `it` assertions. Also for this particular one, since we are analyzing an output that happens to be an object, it would look better if we analyze every property of said object individualy, and the presence of that property with a` .toHaveProperty` assertion.

Reply:
Fixed by adding `describe` block and use `it` rather than `test`.